config/initializers/bulkrax.rb

```diff
@@ -50,8 +50,6 @@
       "title" => { from: ["title"] },
       "types" => { from: ["type"], split: /\s*[;]\s*/, parsed: true },
       "remote_files" => { from: ['thumbnail_url'], parsed: true },
-      'parents' => { from: ['parents'], related_parents_field_mapping: true },
-      'children' => { from: ['children'], related_children_field_mapping: true }
```

Comment:
these two settings don't do anything because the oai feed doesn't have a property that correlates to a unique identifier for the collection, on the work.